src/app/app.component.ts

```diff
@@ -44,6 +44,7 @@ export class AppComponent implements OnInit {
     }
     this.allItems.unshift({
       // NOTE: The line below is confusing, why description instead of description: description
+      // Its the shorthand syntax introduced in es6
```

Comment:
Thanks for this clarification. It just didn't make sense to me before because I had never seen that syntax before. 
But it makes sense now. Considering the fact that it enforces DRY.
```suggestion
      // NOTE: description below is shorthand for description: description
      // It's the shorthand syntax introduced in es6
```